Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.xaml

```diff
@@ -101,6 +103,13 @@
         <Setter Property="BorderThickness" Value="1" />
         <Setter Property="IsTabStop" Value="False" />
         <Setter Property="IsExpanded" Value="True"/>
+        <Setter Property="HeaderTemplate">
+            <Setter.Value>
+                <DataTemplate>
+                    <TextBlock Text="{Binding}" Margin="4,0,0,0" VerticalAlignment="Center"/>
```

Comment:
Don't forget the HorizontalAlignment either, default is Stretch not Left, as it was before. :)

Reply:
But TextBlock aligns to the left, so it's handled

Reply:
![image](https://user-images.githubusercontent.com/24302614/32684759-ceecc2a0-c63b-11e7-9687-e3a11fb145f6.png)

Well, the `HorizontalTextAlignment` is Left but the `HorizontalAlignment` is still Stretch.  Practically though, it won't make a difference unless someone was using the VisualTree to fiddle with stuff like Width; which for this case now, is to just use their own Header.